.github/workflows/molecule-actions.yml

```diff
@@ -29,6 +29,8 @@ jobs:
           - core/time
           - addons/clustershell
           - addons/users_basic
```

Comment:
Can you move this a few lines above with core roles? I tried to order alphabetically per role category priority (core/advanced-core/addons):

- core/dns_server
- core/log_client
- core/log_server
- core/time